operators/multiclusterobservability/manifests/base/alertmanager/alert_rules.yaml

```diff
@@ -37,9 +37,10 @@ data:
           annotations:
             summary: "There is a policy report violation with a {{ $labels.severity }} severity level detected."
             description: "The policy: {{ $labels.policy }} has a severity of {{ $labels.severity }} on cluster: {{ $labels.cluster }}"
-          expr: sum(policyreport_info) by (managed_cluster_id, category, clusterID, policy, severity) > 0
+          expr: sum(policyreport_info * on (managed_cluster_id) group_left (cluster, hub_cluster_id) label_replace(acm_managed_cluster_labels, "cluster", "$1", "name", "(.*)")) by (cluster, hub_cluster_id, managed_cluster_id, category, clusterID, policy, severity) > 0
           for: 1m
           labels:
+            clusterID: "{{ $labels.managed_cluster_id }}"
```

Comment:
I am assuming you have checked that real alerts add the other labels:
- cluster
- hub_cluster_id.

This is neat and slick.

Where I hesitate a little is if we just run this query in line 40, the result is confusing on the columns:
- clusterID, hub_cluster_id and managed_cluster_id.
That confusion is sorted out only when we look at the label swapping.
That gives me a little pause.